static/js/web-install.js

```diff
@@ -196,13 +200,15 @@ async function flashRelease(setProgress) {
     }
 
     setProgress("Flashing release...");
+    safeToLeave = false;
     await device.flashFactoryZip(blob, true, reconnectCallback,
         (action, item, progress) => {
             let userAction = fastboot.USER_ACTION_MAP[action];
             let userItem = item === "avb_custom_key" ? "verified boot key" : item;
             setProgress(`${userAction} ${userItem}...`, progress);
         }
     );
+    safeToLeave = true;
```

Comment:
I think the best way to do this is with `try {} finally { safeToLeave = true; }` to make sure it always runs and in the smallest scope possible right after setting it to `false`.

Reply:
Sorted.